feature/account/setup/src/main/kotlin/app/k9mail/feature/account/setup/ui/specialfolders/SpecialFoldersViewModel.kt

```diff
@@ -163,8 +163,10 @@ class SpecialFoldersViewModel(
         viewModelScope.coroutineContext.cancelChildren()
         updateState {
             it.copy(
+                isLoading = true,
                 error = null,
             )
         }
+        onLoadSpecialFolderOptions()
```

Comment:
Should be validated in the test: `should show form when OnRetryClicked event received`

Reply:
Done.